src/nCrCatalan/index.js

```diff
@@ -0,0 +1,18 @@
+/* Nth Catalan Number
+ * Function: nCrCatalan()
+ */
+ const validate = require('../validation/non-negative-integer');
+ const nCr = require('../nCr/index.js');
+ function nCrCatalan(num)
+{
+    if (num <= 1){return 1;}
+    let res = 0;
+    for (let i=0; i<num; i++){
+        res += nCr(i)*nCr(num-i-1);	
```

Comment:
Did you test this?
nCr function takes 2 numbers as argument, you have passed only one.